src/patches/qfluent_navigation_patch.py

```
@@ -25,7 +25,7 @@ def install_qfluent_navigation_patch():
         return
     try:
         from qfluentwidgets.components.navigation.navigation_panel import NavigationPanel
-    except Exception as exc:
+    except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)